nni/contribute/distillation/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
nni/contrib/distillation

Reply:
On a second thought, `nni/contrib/distill` might be even better, because it's shorter.

But since `nni.compression` is already long, you can also keep the name `distillation` if you want to stick to the tradition.